tests/test_evals/fgm_attack.py

```diff
@@ -0,0 +1,107 @@
+"""
```

Comment:
this file has been moved to `tests/evals/fgm_attack.py`. I would remove this from the PR.